lib/Doctrine/ORM/Cache/DefaultCacheFactory.php

```diff
@@ -201,7 +209,7 @@ public function getRegion(array $cache)
             }
 
             $directory = $this->fileLockRegionDirectory . DIRECTORY_SEPARATOR . $cache['region'];
-            $region    = new FileLockRegion($region, $directory, $this->regionsConfig->getLockLifetime($cache['region']));
+            $region    = new FileLockRegion($region, $directory, (string) $this->regionsConfig->getLockLifetime($cache['region']));
```

Comment:
seems like the 3rd param should be an int instead of a string, it solves everything without changing the code

Reply:
Doesn't switching to an int be a breaking change though?

Reply:
Not sure how public FileLockRegion is. However, your PR already changed the type to numeric-string. In for a penny... :)

Reply:
`numeric-string` is still a string, and yes, switching to `int` would be a breaking change. Definitely not something you would want to do as part of a patch. As for `FileLockRegion`, it's not marked as internal, so in doubt, let's assume it's very much public.